templates/partials/_md_file_sec.fragment.xml

```diff
@@ -0,0 +1,9 @@
+{% if resource.data_files %}
+  <METS:fileSec>
+    {% for file in resource.data_files %}
+    <METS:file ID="{{ file.id }}" {% if file.group_id %}GROUPID="{{ file.group_id }}"{% endif %} USE="{{ file.use }}"  MIMETYPE="{{ file.ref.mimetype }}" MDID="{{ file.mdid }}">
+      <METS:FLocat LOCTYPE="SYSTEM" LOCREF="../{{ relative_path_or_not(file.ref.locref) }}" />
```

Comment:
Should the leading `../` be removed here if `relative_path_or_not` is handling it?